xyz

```diff
@@ -119,8 +119,14 @@ next_version=$("$dir/node_modules/.bin/semver" -i "$increment" "$version") ||
 message="${message_template//X.Y.Z/$next_version}"
 tag="${tag_template//X.Y.Z/$next_version}"
 
-bold=$(tput bold)
-reset=$(tput sgr0)
+if [[ -t 1 ]] && [[ -n "$TERM" ]] && which tput &>/dev/null; then
+  bold=$(tput bold)
+  reset=$(tput sgr0)
+else
+  bold=""
+  reset=""
```

Comment:
I don't think the three lines above are necessary. Won't `bold` and `reset` default to empty string if not defined?